generators/add-action/generic/index.js

```diff
@@ -27,6 +27,7 @@ class GenericGenerator extends ActionGenerator {
 
```

Comment:
Also could you handle delete action?

I think that unit test files are not deleted neither (missed that in the passed). Could you delete unit and e2e test files for a given action when running delete-action